utils/details.js

```diff
@@ -364,10 +366,13 @@ ${
       })
     } catch (error) {
       // add a regular comment if we can't add a review comment
-      await octokit.rest.issues.createComment({
+      //docs.github.com/rest/pulls/comments#create-a-review-comment-for-a-pull-request
+      await octokit.request('POST /repos/{owner}/{repo}/pulls/{pull_number}/comments', {
```

Comment:
I could be off base here as I can't say I've ever used this endpoint on the API, but the docs do say if you don't specify the `line` property, you must specify `subject_type: file` so the comment ends up on the file itself and the API doc doesn't define a default value.

Reply:
Great catch, will fix.